telluric/util/raster_utils.py

```diff
@@ -83,3 +88,243 @@ def convert_to_cog(source_file, destination_file, resampling=Resampling.gauss):
             rasterio_sh.copy(temp_file, destination_file,
                              COPY_SRC_OVERVIEWS=True, tiled=True,
                              compress='DEFLATE', photometric='MINISBLACK')
+
+
+def calc_transform(src, dst_crs=None, resolution=None, dimensions=None,
```

Comment:
I guess this comes from rasterio because of https://github.com/mapbox/rasterio/pull/1409. Could we add a comment to a) acknowledge attribution and the rasterio license and b) state that when the pull request is merged we will be able to drop this code?

Reply:
I have realized that this functionality what I proposed in rasterio's PR is not needed for us here. In fact this *calc_transform* function is part of rasterio's [rasterio.rio.warp](https://github.com/mapbox/rasterio/blob/2f869c4ebbae90092aae53f9eefcc8a480b7aaf4/rasterio/rio/warp.py#L67) console utility. I splitted original *rasterio.rio.warp* into two functions in telluric: *raster_utils.calc_transform* and *raster_utils.warp*. This was done in this way because in *GeoRaster.reproject* we call *raster_utils.calc_transform* for calculating transformation stuff for in-memory rasters but the reprojection itself is performed by *GeoRaster._reproject* (not by *raster_utils.warp* as for non-in-memory rasters).